backend/api/models/ReviewModel.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../../config/knexConfig");
+
+module.exports = {
+  getAll_reviews,
+  add
+};
+
+function getAll_reviews() {
+  return db("reviews");
+}
+
+function add(review) {
```

Comment:
might make senses to keep our function naming consistent across models. 

type_table --> getAll_reviews // get_review // add_review // update_user etc etc // delete_user

Reply:
Agreed, this is something I've been meaning to talk with the group about to see how we like to keep these naming conventions